### PR TITLE
Add documentation on MySQL connection string docs

### PR DESCRIPTION
**Pull request by scouvreur**
This PR adds documentation about an error in a mysql connection which if not configured correctly can cause an issue when spinning up the airflow docker image. Closes issue #14104.

### CODE REVIEW

File: airflow/config_templates/config.yml [hunk 1]
```diff
@@ -64,7 +64,9 @@
       description: |
         The SqlAlchemy connection string to the metadata database.
         SqlAlchemy supports many different database engine, more information
-        their website
+        their website. You will need a user and password provided in the database
```
Comment (mik-laj) [inline airflow/config_templates/config.yml]: It seems to me that it is not worth describing individual drivers here, but we can add a link to the documentation describing ttis in detail.
http://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#database-uri

Comment (scouvreur) [reply airflow/config_templates/config.yml]: Thanks for the review @mik-laj - just changed it in the recent commit. Apologies for the delay !

**Comment by mik-laj** [comment]
Static checks are sad. Can you fix it?

**Comment by scouvreur** [comment]
Sure @mik-laj - will fix the checks !

**Comment by scouvreur** [comment]
Static checks are good now, looking at the failure on `Postgres9.6,Py3.6`

**Comment by mik-laj** [comment]
Please ignore these errors.

**Comment by scouvreur** [comment]
Thanks @mik-laj, just tested locally and it passed - is there anything else you need on this PR ?

**Comment by scouvreur** [comment]
Hey @mik-laj - is there anything else you need from me on this PR ?

**Review by mik-laj — APPROVED** [review]

**Pull request merged**